src/collision/CollisionManager.js

```diff
@@ -100,8 +97,11 @@ exports = Class(function(supr) {
   this.start = function(collisionCheckID) {};
 
   this.removeCollisionsContaining = function(collider) {
-    if (this._collidersToRemove.indexOf(collider) === -1 ) {
-      this._collidersToRemove.push(collider);
+    if (this._checksToRemove.indexOf(collider) === -1) {
```

Comment:
This won't work here, since you're checking if the array has the collider, not its collisions.